app/assets/javascripts/search.js

```diff
@@ -31,7 +31,7 @@
       return searchResultData;
     },
     enableLiveSearchCheckbox: function ($searchResults) {
-      if ($searchResults.length > 0) {
+      if ($searchResults.length >= 0) {
```

Comment:
This discussion was just had in person. Expect the commit message and PR message to be updated to make this clearer. In the mean time, I'm going to close this PR.